tests/test_callbacks.py

```diff
@@ -10,8 +10,11 @@
     CheckpointCallback,
     EvalCallback,
     EveryNTimesteps,
+    StopTrainingOnMaxEpisodes,
     StopTrainingOnRewardThreshold,
 )
+from stable_baselines3.common.cmd_util import make_vec_env
+from stable_baselines3.common.vec_env import SubprocVecEnv
```

Comment:
This could be DummyVecEnv (API is same but no need to hassle with processes).

Reply:
This was required to make testes for models that support multiple envs (A2C and PPO).

Reply:
@araffin Any comment?

Reply:
https://stable-baselines3.readthedocs.io/en/master/guide/vec_envs.html

I agree with @Miffyli

Reply:
But then I would have to drop the code for testing multiprocessing in A2C and PPO cases.

Reply:
You can test with multiple environments but without processes (usually faster), this is what the `DummyVecEnv` is for

see our tutorial https://github.com/araffin/rl-tutorial-jnrr19/tree/sb3 for more details

Reply:
Oh sorry, I did not fully understand that. I thought that `DummyVecEnv` could only be used for single envs. I will change the code accordingly, thank you! =)